commands/update.js

```diff
@@ -0,0 +1,58 @@
+const exec = require('child_process').exec;
```

Comment:
This might be possible as per http://stackoverflow.com/questions/15957529/can-i-install-a-npm-package-from-javascript-running-in-node-js/15957574#15957574

but probably it's not a good idea given that npm will install many other dependencies (we need to do `npm install --save install`).